lib/configuration_methods.rb

```diff
@@ -69,6 +69,25 @@ def secret_token
       end
     end
 
+    def twofa_encryption_key
+      if heroku?
+        return ENV['TWOFA_ENCRYPTION_KEY'] if ENV['TWOFA_ENCRYPTION_KEY']
+        warn "FATAL: Running on Heroku with TWOFA_ENCRYPTION_KEY unset"
+        warn "       Run heroku config:add TWOFA_ENCRYPTION_KEY=#{SecureRandom.hex(32)}"
+        Process.exit(1)
+      else
+        key_file = File.expand_path(
+          '../config/initializers/twofa_encryption_key.rb',
```

Comment:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.